src/OpenTelemetry/Instrumentation/ListenerHandler.cs

```diff
@@ -17,29 +17,63 @@
 
 namespace OpenTelemetry.Instrumentation
 {
+    /// <summary>
+    /// ListenerHandler base class.
+    /// </summary>
     public abstract class ListenerHandler
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ListenerHandler"/> class.
+        /// </summary>
+        /// <param name="sourceName">The name of the <see cref="ListenerHandler"/>.</param>
         public ListenerHandler(string sourceName)
         {
             this.SourceName = sourceName;
         }
 
+        /// <summary>
+        /// Gets the name of the <see cref="ListenerHandler"/>.
```

Comment:
Just double-checking, it's the name of the ListenerHandler or the name of the source the ListenerHandler will use to listen?

Reply:
I'm not sure about that. Probably the name of the source.